packages/drivers/odsp-driver/src/odspDriverUrlResolverForShareLink.ts

```diff
@@ -74,12 +69,12 @@ export class OdspDriverUrlResolverForShareLink implements IUrlResolver {
         private readonly appName?: string,
     ) {
         this.logger = createOdspLogger(logger);
-        if (shareLinkFetcherProps) {
-            this.shareLinkFetcherProps = {
-                ...shareLinkFetcherProps,
-                tokenFetcher: this.toInstrumentedTokenFetcher(this.logger, shareLinkFetcherProps.tokenFetcher),
-            };
-        }
+         if (shareLinkFetcherProps) {
+             this.shareLinkFetcherProps = {
+                 ...shareLinkFetcherProps,
+                 tokenFetcher: shareLinkFetcherProps.tokenFetcher,
+             };
+         }
```

Comment:
nit - extra space showed up
```suggestion
        if (shareLinkFetcherProps) {
            this.shareLinkFetcherProps = {
                ...shareLinkFetcherProps,
                tokenFetcher: shareLinkFetcherProps.tokenFetcher,
            };
        }
```